packages/chrome/src/styled/sheet/StyledSheetCloseButtonContainer.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_close';
+
+const placementStyles = (props: ThemeProps<DefaultTheme>) => {
+  const placement = props.theme.rtl ? 'left' : 'right';
+
+  return css`
+    top: ${props.theme.space.base * 3}px;
+    ${placement}: ${props.theme.space.base * 1}px;
+  `;
+};
+
+export const StyledSheetCloseButtonContainer = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<ThemeProps<DefaultTheme>>`
+  display: block;
+  position: absolute;
```

Comment:
Any reasoning to not use `display: flex` for positioning?

Reply:
I think either would work - just realized a `div` has a default `display: block`. Any particular reason to use `flex` in this instance?

Reply:
No hard requirements here. Flex has more flexibility (no pun intended) for dynamic layouts and RTL capabilities with `flex-direction: row-reverse` and could potentially scale nicely with whatever `theme` variables might be used.

Reply:
Agreed! I think for the button, it's only going to have a single child (SVG icon) and can get away with the default display for that reason I believe. It's also positioned as `absolute` against the Sheet layout.

Reply:
Good call on this, after reverting to a custom button, `flex` layout was the right choice here!